e2e/images_test.go

```diff
@@ -0,0 +1,53 @@
+package e2e
+
+import (
+	"fmt"
+	"path/filepath"
+	"regexp"
+	"testing"
+
+	"gotest.tools/assert"
+	"gotest.tools/fs"
+	"gotest.tools/icmd"
+)
+
+var (
+	reg      = regexp.MustCompile("Digest is (.*).")
+	expected = `REPOSITORY             TAG    APP NAME
+%s        push-pull
+a-simple-app           latest simple
+b-simple-app           latest simple
+`
+)
+
+func TestImageList(t *testing.T) {
+	runWithDindSwarmAndRegistry(t, func(info dindSwarmAndRegistryInfo) {
+		cmd, cleanup := dockerCli.createTestCmd()
```

Comment:
Why not to use `cmd := info.configuredCmd`? And this way the `cleanup` is already called by `runWithDindSwarmAndRegistry`

Reply:
I blindly copied the code from another test... Thanks!